scripts/create-cba-export.js

```diff
@@ -20,7 +33,16 @@ collect(input, (i) => {
             feature.properties['vpromm_id'] = feature.properties.or_vpromms;
             feature.properties['section_id'] = feature.properties.or_section;
             feature.properties['link_id'] = feature.properties.or_link;
-            console.log(headers.map(h => feature.properties[h]).join(','));
+            let row = [];
+            row = headers.map((h) => {
+                if (feature.properties.hasOwnProperty(h)) {
+                    return feature.properties[h];
+                } else {
+                    return undefined;
```

Comment:
@geohacker maybe better to return an empty string here instead of `undefined` ?

Reply:
Yeah makes sense.